easybuild/easyconfigs/v/vip/vip-v3.0.1-foss-2018b.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Tarball'
+
+name = 'vip'
+version = 'v3.0.1'
+
+homepage = 'https://github.com/molgenis/vip/'
+description = """Variant Interpretation Pipeline"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [('http://github.com/molgenis/%s/archive/' % (name))]
+sources = [('%s.tar.gz' % (version))]
+checksums = ['408508bc37c85cd24558f8294000cd4fc491e9d8d904d0727f0a058fba1efd3d']
+
+#
+# set default singularity image folder
+#
+postinstallcmds = [
+    'cd %(installdir)s/singularity && ln -s /apps/data/vip/%(version)s/sif sif',
```

Comment:
If I understand correctly the module will now depend on `/apps/data/vip/%(version)s/sif`, which is not deployed with this repipe making the recipe incomplete... Not sure if this is the way to go. In addition building the sif is not covered by the recipe. Did you see https://docs.easybuild.io/en/latest/Containers.html? If you simply want to deploy a pre-built sif, you could add it as an asset to a release in GitHub and tell EasyBuild to download it by adding it to the list of "sources"

Reply:
We haven't decided yet where to build and host containers so this is a temporary solution that will be removed in future versions. The module works without issues if the symlink points to a folder that doesn't exist, but if the symlink can be removed then it will be used as a default.